flytekit/annotated/task.py

```diff
@@ -208,6 +221,36 @@ def execute(self, **kwargs) -> Any:
     def native_interface(self) -> Interface:
         return self._native_interface
 
+    def get_registerable_entity(self, settings: ControlPlaneSettings) -> _registerable.RegisterableEntity:
+        if self._registerable_entity is not None:
+            return self._registerable_entity
+
+        from flytekit.configuration import internal as _internal_config
+        args = [
```

Comment:
I think we should make the get_command an overridable function so that spark can easily override it?

Reply:
Sure sounds good.  Currently the entire get_registerable_entity function is overridable.  We can change this to any combination of get container, get command, get entity, etc.